Project.toml

```diff
@@ -12,8 +12,18 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
 AbstractFFTs = "1.5"
+Aqua = "0.8"
 FFTW_jll = "3.3.9"
+LinearAlgebra = "<0.0.1, 1"
 MKL_jll = "2019.0.117, 2020, 2021, 2022, 2023, 2024"
 Preferences = "1.2"
 Reexport = "0.2, 1.0"
+Test = "<0.0.1, 1"
 julia = "1.6"
+
+[extras]
+Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[targets]
+test = ["Aqua", "Test"]
```

Comment:
Why need to duplicate test dependencies?

Reply:
Fixed. I had assumed incorrectly that Aqua requires this.